gitguard.py

```diff
@@ -249,3 +254,128 @@ def get_commit_history(repo_link, author_name=None, start=None, end=None, path=N
         commit['commit_message'] = commit_history[i]["commit"]["message"]
         history.append(commit)
     return history
+
+def _remove_readonly(func, path, excinfo):
+    """
+    Helper function to remove change read-only file status to editable
+    Source: http://stackoverflow.com/questions/1889597/deleting-directory-in-python
+    """
+    os.chmod(path, stat.S_IWRITE)
+    func(path)
+
+def _clone_repo(repo_link, destination=None):
+    """
+    Helper function to clone a repo to destination. If folder already exists, delete all version
+    
+    Args:
+        repo_link:      owner/repo format
+        destination:    local file path, set to default if not supplied
+
+    Return:
+        reference to the cloned repo
+    """
+    if not destination:
+        destination = "%s%s%s" % ("C:/", repo_link, "-gitguard")
```

Comment:
Use relative path instead. Just make a folder inside the current subdirectory. User might not run it from windows.